packages/library-base/src/commonMain/kotlin/io/realm/kotlin/exceptions/RealmException.kt

```diff
@@ -9,7 +9,7 @@ package io.realm.kotlin.exceptions
  */
 public open class RealmException : RuntimeException {
     public constructor() : super()
-    public constructor(message: String) : super(message)
-    public constructor(message: String, cause: Throwable) : super(message, cause)
-    public constructor(cause: Throwable) : super(cause)
+    public constructor(message: String?) : super(message)
+    public constructor(message: String?, cause: Throwable) : super(message, cause)
```

Comment:
`cause` is also nullable https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-runtime-exception/#constructors